packages/kbn-optimizer/limits.yml

```diff
@@ -75,7 +75,7 @@ pageLoadAssetSize:
   watcher: 43598
   runtimeFields: 41752
   stackAlerts: 29684
-  presentationUtil: 84606
+  presentationUtil: 176421
```

Comment:
Does this really need increased and not reduced? If so, something seems off. 

We really want to keep plugins [under 100kB](https://github.com/elastic/kibana/issues/95853) if possible.

Reply:
Wow, yeah that was a pretty major increase. Basically what happened here is that components and dependencies which were previously only used _inside_ the presentation util, by async imported components now needed to be exported as shared components for use in the controls plugin.

To reduce the bundle size back under 100kb, I had to wrap all of the exported components in React.lazy. The bundle is still larger, but not so massively large.

There is more to do to further shrink the bundle size, especially figuring out how to remove the flights storybook fixture data from the bundle. I have confirmed that it isn't imported or exported anywhere, but it is under a directory within `extraPublicDirs` so it can be imported to storybooks in other plugins.

Reply:
Thanks so much for looking into this.